src/bin/visit.c

```diff
@@ -332,12 +378,56 @@ VisItLauncherMain(int argc, char *argv[])
         }
         else if(ARG("-help"))
         {
-            printf("%s", usage);
+            string usageFile(GetUsageTextDir());
+            usageFile += "visitusage.txt";
+            std::ifstream inFile;
+            inFile.open(usageFile);
+            std::stringstream strStream;
+            strStream << inFile.rdbuf();
+            string usageText = strStream.str();
+            inFile.close();
+            // add the windows-specific text
+            usageText += string(windows_usage);
+
+#ifdef VISIT_WINDOWS_APPLICATION
+            // Create a console and redirect i/o to it.
+            // A MessageBox is good only for small amounts of text.
+            if (CreateNewConsole(1024))
+            {
+                cerr << usageText.c_str() << endl;
+                //cerr << "NOTE: For a more complete list of options, use '-fullhelp'." << endl;
```

Comment:
Is `-fullhelp` still disabled on Windows?

Reply:
Correct, -fullhelp still disabled.
Still awaiting more feedback on #5101.  Thought it good to get the shorter help in and working correctly on Windows to address the customer reported bug.